blockchain_parser/transaction.py

```diff
@@ -49,6 +58,18 @@ def __init__(self, raw_hex):
             offset += output.size
             self.outputs.append(output)
 
+        # currently only supports parsing of segwit transactions without error
+        # not actually saving their witnesses
+        if self.is_segwit:
+            self._offset_before_tx_witnesses = offset
+            for i in range(self.n_inputs):
+                tx_witnesses_n, varint_size = decode_varint(raw_hex[offset:])
+                offset += varint_size
+                for j in range(tx_witnesses_n):
+                    component_length, varint_size = decode_varint(raw_hex[offset:])
```

Comment:
@alecalve If you want to store the witnesses, this would be the place to do it. I don't need to for my purposes, but it would be great if the library did. I've refrained for the moment because I wasn't sure how you wanted to represent them (I'm not totally sure what format they take), but I can add that functionality if desired, or you are welcome to :)